charts/spire/charts/tornjak-frontend/templates/ingress.yaml

```diff
@@ -1,13 +1,19 @@
 {{- if .Values.ingress.enabled -}}
+{{- $ingressControllerType := include "spire-lib.ingress-controller-type" (dict "global" .Values.global "ingress" .Values.ingress) }}
 {{- $fullName := include "tornjak-frontend.fullname" . -}}
+{{- $annotations := deepCopy .Values.ingress.annotations }}
+{{- if eq $ingressControllerType "ingress-nginx" }}
+{{-   $_ := set $annotations "nginx.ingress.kubernetes.io/ssl-redirect" "true" }}
+{{-   $_ := set $annotations "nginx.ingress.kubernetes.io/force-ssl-redirect" "true" }}
```

Comment:
The set of annotations here is less than server. Is that intended?

Reply:
yeah. server has to have passthrough to deal with its spire issued certificates. This has regular certs so doesn't need it. Good catch to double check though.

Reply:
Thanks for confirming.